tests/integration/targets/dcnm_network/tests/dcnm/replaced.yaml

```diff
@@ -115,14 +115,14 @@
       deploy: true
   register: result
 
-- name: Query fabric state until networkStatus transitions to DEPLOYED state
+- name: Query fabric state until networkStatus transitions to NA state
   cisco.dcnm.dcnm_network:
     fabric: "{{ test_fabric }}"
     state: query
   register: query_result
   until:
-    - "query_result.response[0].parent.networkStatus is search('DEPLOYED')"
-    - "query_result.response[1].parent.networkStatus is search('DEPLOYED')"
+    - "query_result.response[0].parent.networkStatus is search('NA')"
```

Comment:
Why are we changing this from `DEPLOYED` to `NA`?

Reply:
In this particular test we are  not attaching any switches to network, so the switch detail will never have status as deployed. It will be be just NA.